src/_components/Footer.jsx

```diff
@@ -87,65 +84,39 @@ export default function Footer() {
                 </div>
               </div>
             </div>
-            <div className="grid grid-cols-2 gap-20 sm:gap-60 sm:grid-cols-3 ">
+
+            <div className="grid grid-cols-1 md:grid-cols-2 gap-x-0 sm:gap-x-1 lg:gap-x-2">
               <div>
-                <h2 className="mb-6  text-md font-semibold font- poppins text-white uppercase ">
+                <h2 className="mb-6 text-md font-semibold font-poppins text-white uppercase">
                   General
                 </h2>
-                <ul className="text-gray-300  font-medium">
-                  <li className="mb-4 font- poppins">
+                <ul className="text-gray-300 font-medium">
+                  <li className="mb-4 font-poppins">
                     <a href={mission_url} className="hover:underline">
                       Mission
                     </a>
                   </li>
-                  <li className="mb-4 font- poppins">
+                  <li className="mb-4 font-poppins">
                     <a href={projects_url} className="hover:underline">
                       Projects
                     </a>
                   </li>
-                  <li className="mb-4 font- poppins">
+                  <li className="mb-4 font-poppins">
                     <a href="mailto:sit.blueprint@gmail.com">Contact</a>
                   </li>
                 </ul>
               </div>
               <div>
-                <h2 className="mb-6 text-md font-semibold font- Poppins uppercase text-white">
-                  Students
-                </h2>
-                <ul className="text-gray-300 font-medium">
-                  <li className="mb-4 font- poppins">
-                    <a href={community_url} className="hover:underline ">
-                      Get Involved
-                    </a>
-                  </li>
-                  <li className="mb-4 font- poppins">
-                    <a href={process_url} className="hover:underline">
-                      Process
-                    </a>
-                  </li>
-                  <li className="mb-4 font- poppins">
-                    <a href={faq_url} className="hover:underline">
-                      FAQ
-                    </a>
-                  </li>
-                </ul>
-              </div>
-              <div>
-                <h2 className="mb-6 text-md font-semibold font- poppins text-white uppercase">
-                  Non-Profits
+                <h2 className="mb-6 text-md font-semibold font-poppins uppercase text-white">
+                  Students & Non-Profits
```

Comment:
Could we change `General' and `Students & Non-Profits` to be online on mobile? I think it would look better if you had just two columns for each.

Reply:
I agree i think it would be nicer to have them be inline so that it would be a 2 column layout with `General` and `Students & Non-profits` on the right